buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -68,6 +68,13 @@ class ExampleRunnerTask extends JavaExec {
   // The base package for all examples.
   private String basePackage = 'com.google.ads.googleads.examples.'
 
+  // Hints for failed executions.
+  private final String exampleInvocation = 'E.g. ' +
```

Comment:
Nit: Use "For example" instead of "E.g.", per our tech writers:

https://developers.google.com/style/word-list#letter-e

Reply:
Fixed.